src/interp.rs

```diff
@@ -130,7 +130,7 @@ fn eval_assign<'a, 'tcx>(_block: BasicBlock,
     }
 }
 
-/// Each BasicBlocks is succeeded by a terminator, which encapsulates control
+/// Each BasicBlock is succeeded by a terminator, which encapsulates control
```

Comment:
Do you want to put `BasicBlock` in backquotes so it gets formatted as code?

Reply:
Sure